src/content/components/Preferences/Preferences.js

```diff
@@ -1,6 +1,8 @@
 import React from "react";
```

Comment:
Conflict with #263 - needs to be rebased and revised to work w/ your new settings page